backend/app/services/clip_builder.py

```
@@ -308,8 +308,8 @@ def _collect_recordings(
                     if p.stat().st_size < 1024:
                         p.unlink(missing_ok=True)
                         logger.debug("Cleaned up failed conversion file: %s", p.name)
-                except Exception:
-                    pass
+                except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)